src/txn/txn_rollback_to_stable.c

```diff
@@ -148,6 +191,31 @@ __rollback_row_add_update(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW *rip,
     return (ret);
 }
 
+/*
+ * __rollback_col_ondisk_fixup_key --
+ *     Allocate tombstone and calls function add update to head of insert list.
+ */
+static int
+__rollback_col_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_REF *ref, WT_COL *cip,
+  wt_timestamp_t rollback_timestamp, bool replace, uint64_t recno)
+{
+    WT_DECL_RET;
+    WT_UPDATE *upd;
+
+    WT_UNUSED(rollback_timestamp);
+    WT_UNUSED(replace);
+    WT_UNUSED(page);
+
+    /* Allocate tombstone to update to remove the unstable value. */
+    WT_ERR(__wt_upd_alloc_tombstone(session, &upd, NULL));
```

Comment:
The first statement can be changed into `WT_RET`.

Reply:
Is that because the above function would then free upd?

Reply:
Yes. Another approach is to set `upd = NULL` as `__wt_free` checks for NULL.

Reply:
Ah okay, thanks!